.github/actions/abort-release-candidate-v1/action.yml

```diff
@@ -12,6 +12,9 @@ inputs:
     description: 'The docs repository where the issue for the release candidate is located'
     default: ''
     required: false
+  version:
```

Comment:
I'm not entirely sure we want this. The abort-release-candidate action should be able to infer the version of the RC it's trying to abort. I think there is a something wrong with _how_ we are not checking out the `release` branch? 

I would imagine if you run `create-release-candidate`, that PR gets merged into `release` and now release holds a bumped version of e.g. `1.1.0`. Checking out `origin/release` and running  this [snippet](https://github.com/dequelabs/axe-api-team-public/pull/172/files#diff-58d5337330f7e542de2ed43b7f979a91c8ea07477f1397b364e19e9816d5ff48L31-L37) should pick up the new bumped version `1.1.0`

Reply:
As we discussed with @dequejenn we should be able to abort RC whenever we want.
In the first case, we want to abort RC **before** merging `release-YYYY-MM-DD -> release`. In this case, we provide the exact version because `lerna.json` or `package.json` files have the old version number.
In the second case, we want to abort RC **after** merging into the `release` branch. In this case we will take the version number from `lerna.json` or `package.json` files